addon-test-support/@ember/test-helpers/dom/trigger-key-event.js

```diff
@@ -0,0 +1,23 @@
+import { merge } from '@ember/polyfills';
+import triggerEvent from './trigger-event';
+
+/**
+ * @public
+ * @param selector
+ * @param type
+ * @param keyCode
+ * @param modifiers
```

Comment:
the parameters are missing type information

Reply:
Good catch